x/pki/keeper/msg_server_propose_add_x_509_root_cert.go

```diff
@@ -91,6 +91,11 @@ func (k msgServer) ProposeAddX509RootCert(goCtx context.Context, msg *types.MsgP
 	// store proposed certificate
 	k.SetProposedCertificate(ctx, proposedCertificate)
 
+	_, isFound := k.GetRejectedCertificate(ctx, proposedCertificate.Subject, proposedCertificate.SubjectKeyId)
```

Comment:
Why can't we just delete without checking?
Why should we delete rejected certificate once proposed?
Shoukd we able to propose a rejected certificate again?